app/views/images/show.html.erb

```diff
@@ -20,24 +20,14 @@
     <td><%= @image.size %></td>
   </tr>
   <tr>
-    <td><strong>Height </strong></td>
-    <td><%= @image.height %></td>
-  </tr>
-  <tr>
-    <td><strong>Width </strong></td>
-    <td><%= @image.width %></td>
-  </tr>
-  <tr>
     <td><strong>Alt text </strong></td>
     <td><%= @image.alt_text %></td>
   </tr>
 </table>
 
-<a data-lightbox="image-1" data-title="<%= @image.alt_text %>"
-   href="<%= base_src + @image.file_name %>">
-   <img src="<%= base_src + @image.file_name %>" alt="<%= @image.alt_text %>"
-        style="max-width: 600px; max-height: 600px;" />
-</a>
+<div id="osd-viewer" style="width: 600px; height: 600px;"></div>
```

Comment:
Sure, thanks.  This was an oversight.

It reminds me, though, that the dimensions for the viewer in images/show were different than those in sources/show, so I may need to make some adjustments -- or maybe not; I'll see.

It's good to take these styles out of the HTML so that we can eventually get the app to display properly on a mobile device with alternate media stylesheets.